### ISSUE TITLE
Setting colors for the theme

### ISSUE THREAD

**Issue by nabihahossain** [miguelcobain/ember-paper#381]
I am importing my own .scss file to override some of the default settings in ember-paper.scss. I want to use different shades of the color maps. 

How do I select the '800' version of the color 'indigo' for example?

**Comment by DanChadwick** [comment]
Something like this in your scss file:
```
.ic-sort {
  position: absolute;
  top: -5px;
  right: 0;
  color: color($accent, 'A100') !important;
}
```

Normally you would limit yourself to `$primary`, `$accent`, or `$warn`, but you can use any other color name in the palette.

**Comment by nabihahossain** [comment]
thanks for responding @DanChadwick . and how can I set my $primary to be the '800' hue of 'indigo?'

**Comment by DanChadwick** [comment]
The primary color is a set of color (tints and shades) with the same hue. For a given specific use, the component's CSS picks the variation within the color. In your example, `$primary` is `"indigo"`, and `800` is a shade that a particular color want's to override.

If you don't like the color specified by Material Design, you can write your own CSS to override the `ember-paper` CSS (which implements MD via the Angular Material stylesheets). You can use the color function as shown above to substitute `800` where otherwise the stylesheets might have used say `400`.

The idea is that you pick the hue (i.e. `$primary`) and MD uses the right (according to google) shade and tint of that hue for every possible use. 

TL;DR: It's easy to pick a hue and let ember-paper do the "right" thing. Want complete control over every color and every use? Break out your CSS skills because you have a ton of CSS to override.

**Comment by DanChadwick** [comment]
Feel free to reopen if you have a follow-up question.
